packages/lxd/src/cluster/index.coffee.md

```diff
@@ -126,18 +137,32 @@ containers:
                       description: '''
                       Enable SSH connection.
                       '''
-              required: ['image']
+              # required: ['image']
           'networks':
             type: 'object'
             default: {}
             patternProperties: '.*':
               $ref: 'module://@nikitajs/lxd/src/network#/definitions/config/properties/properties'
           'prevision':
             typeof: 'function'
+            description: '''
+            A nikita action called before the container's creation.
+            '''
+          'prevision_container':
+            typeof: 'function'
+            description: '''
+            A nikita action called for every container before it is created.
+            '''
           'provision':
             typeof: 'function'
+            description: '''
+            A nikita action called after the container's creation.
+            '''
           'provision_container':
             typeof: 'function'
+            description: '''
+            A nikita action called for every container after it is created.
+            '''
```

Comment:
This is already present on master, it shouldn't appear in this merge request. Also, the merge request is about `lxd.file.pull`, I don't see why cluster should be affected at all.

Reply:
It is a mistake. It shouldn't be there, but I needed this commit so I had to fetch it. It wasn't supposed to be in the pull request.